### ISSUE TITLE
Incorrect focus order on CMS fields with multiple content blocks

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10115]
## Description
On the edit screen in the CMS, in the fields where the editor is able to add multiple blocks of content, when a keyboard user expands or collapses a block, the focus is brought back to the top of the widget instead of continuing at the next logical element.

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/183700724-aa3b5948-0ef3-48b1-b0b1-633b8e7d2177.png)

## Accessibility Standard

WCAG version 2.0 A, [Criterion 2.4.3](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html)

## Acceptance Criteria
- [ ] Technical review
- [ ] Change management consulted
- [ ] Implementation ticket created

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by laflannery** [comment]
I am curious if this would possible get fixed on it's own when Drupal get's updated.  I will test again when that happens

**Comment by srancour** [comment]
This is still an issue and should be worked on.
